code/modules/surgery/organ_manipulation.dm

```diff
@@ -95,7 +95,6 @@
 			to_chat(user, "<span class='notice'>There is no room for [I] in [target]'s [parse_zone(target_zone)]!</span>")
 			return -1
 	if(istype(tool, /obj/item/organ/brain/positron))
-		current_type = "posibrain"
```

Comment:
Why was this removed?

Reply:
it conflicted with the surgery, somehow, if i didnt remove it you couldnt place the brain back.